src/components/PullRequestList/index.tsx

```diff
@@ -6,6 +6,19 @@ import CardShimmer from '@/components/Loaders/cardShimmer';
 import useFetch from '@/hooks/useFetch';
 import styles from './PullRequestList.module.scss';
 
+const SCREEN_HEIGHT = globalThis.innerHeight;
+const SCREEN_WIDTH = globalThis.innerWidth;
+const CARD_HEIGHT = 147;
+const MINIMUM_CARD_WIDTH = 300;
+const MAXIMUM_NUMBER_OF_CARDS_HORIZONTALLY = 3;
+const MINIMUM_NUMBER_OF_CARDS_HORIZONTALLY = 1;
```

Comment:
If you look at the CSS styles written for cards in [PullRequestList.module.scss](https://github.com/Real-Dev-Squad/website-status/blob/d852daf809f0fde678228bac369e38243869e72d/src/components/PullRequestList/PullRequestList.module.scss#L33-L36) we can see that the Card's width is `300px` OR `25%` of the `viewport width`.

Using that data we can simplify the logic like this,

```JS
const minCardWidth = 300;
const cardWidth = (
        (screenWidth / 4) > minCardWidth
    )
    ? (screenWidth / 4)
    : minCardWidth;
```

Reply:
I did not understand the use of getting cardWidth when we can get the value(numberofcards) without calculating it (using the min-cardwidth). I have used this method that you told in the PR that I just updated but I think I might have missed out on what you are trying to convey. Please check